ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -160,6 +160,7 @@ ANALYTICS_IDS = {
     'GOOGLE_ANALYTICS_ID': '{{ localsettings.GOOGLE_ANALYTICS_ID }}',
     'PINGDOM_ID': '{{ localsettings.PINGDOM_ID }}',
     'SEGMENT_ANALYTICS_KEY': '{{ localsettings.SEGMENT_ANALYTICS_KEY }}',
+    'HUBSPOT_ID': '{{ localsettings.HUBSPOT_ID }}',
```

Comment:
I think you either have to figure that out or make sure to add `HUBSPOT_ID` to both india and prod (and `dev.yml` in this repo)